lib/core/include/replica.hpp

```diff
@@ -0,0 +1,312 @@
+#ifndef IRODS_REPLICA_HPP
+#define IRODS_REPLICA_HPP
+
+#ifdef RODS_SERVER
+    #define IRODS_FILESYSTEM_ENABLE_SERVER_SIDE_API
+    #include "rsDataObjChksum.hpp"
+#else
+    #undef IRODS_FILESYSTEM_ENABLE_SERVER_SIDE_API
+    #include "dataObjChksum.h"
+#endif
+
+#include "filesystem.hpp"
+#include "irods_exception.hpp"
+#include "key_value_proxy.hpp"
+#include "modDataObjMeta.h"
+#include "objInfo.h"
+#include "query_builder.hpp"
+#include "rcConnect.h"
+
+#include "fmt/format.h"
+
+#include <chrono>
+#include <iomanip>
+#include <string_view>
+
+namespace irods::experimental::replica
+{
+    namespace {
+        namespace fs = irods::experimental::filesystem;
+    } // anonymous namespace
```

Comment:
it is probably best if anonymous namespaces are not used in headers.  i believe for every translation unit that includes this header, this namespace will result in a new symbol.

prefer using the fully qualified names for headers.  there is no good way around this as far as i know.